posts/nachosource/01.organizing-your-react-code.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Organizing your React code: Cohesion and Coupling published: false description:
+tags: 'react, javascript' cover_image:
+---
+
+Have you ever needed to spend a considerable amount of time reviewing your own code (written just a couple of months
+ago) or been assigned to a new project just to find a pile of messy and eternal files?
+
+I'm sure we've all had the feeling that a fix that took a day could've been solved in a fraction of that time by only
+having the code a little more organized.
+
+Consequences of spaghetti code are usually spending days instead of hours solving problems, code repetition,
+inconsistencies between behaviors that should work the same way, inaccurate estimations and, of course, some headaches.
+
+In this series I’ll try to share some concepts that help me when it comes to designing clean and maintainable
+components, architecture and logic.
+
+But first, check this out: the person on the left is a responsible developer who follows the best practices and makes an
+effort to improve his code's legibility when he has the chance. The one on the right is his twin brother, who thinks
+it’s fine to _copy&paste_ the same logic over and over again and concentrate his code in a few files. Choose your cards
+wisely!
+
+![people](./assets/01.organizing-people.png)
+
+## Cohesion and coupling
+
+These two are key concepts that will go along with wherever you code. Since they’ve already been explained way better
```

Comment:
Shouldn't it be: "These two are key concepts that will go along with you wherever you code. Since they’ve already been explained way better"?